console/src/assets/i18n/en.json

```diff
@@ -482,6 +482,7 @@
     "EMAILVERIFIED": "Email verified",
     "NOTVERIFIED": "not verified",
     "PREFERRED_LOGINNAME": "Preferred Loginname",
+    "ISINITIAL": "This user has not been verified yet. Click the button to send a new initialization mail.",
```

Comment:
```suggestion
    "ISINITIAL": "This users email is not verified yet. Select the button to send a new initialization mail.",
```

Reply:
That's what @MattDodsonEnglish used when he reviewed our docs :-)
Maybe on mobile phones, you don't "click" buttons?

Reply:
Exactly :-) . Even on Desktops though: some people prefer to navigate (or must navigate) applications with keyboard.
I stole this from Microsoft: https://github.com/zitadel/zitadel/pull/4380#discussion_r991450499

In the long run, I can't say it's a very important point of technical communication. It assumes a little less about the reader, though :slightly_smiling_face: .

Reply:
But, the best strategy might be to just use the verb that the button represents; in this case it looks like "Send registration link". Generally, avoiding direct references to UI elements makes a doc more future-proof.

Reply:
> But, the best strategy might be to just use the verb that the button represents; in this case it looks like "Send registration link". Generally, avoiding direct references to UI elements makes a doc more future-proof.

I like this approach. Thanks for clarification :blush:

Reply:
@juergrinaldi do you also have an opinion on that?